### DISCUSSION TITLE
Home Assistant OS 14.0.rc2

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3695] [Announcements]
## Home Assistant Operating System

* Enable Cypress M8 USB serial driver for all targets (#3693) @sairon

## Raspberry Pi

* Disable UAS via usb-storage.quirks on RPi for JMicron JMS583 (#3687) @RubenKelevra

## Home Assistant Yellow

* Sync Yellow's DTS with plain CM5 device tree (#3688) @sairon
* Always use NVMe datadisk on Yellow if it's present on first boot (#3686) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.0.rc2'>Home Assistant OS 14.0.rc2</a>.</em>

**Comment by Anto79-ops** [comment]
There  are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.

**Comment by Ladenburg1** [comment]
Unable to install the rc2 on my Raspi5 with an nvme 
After updating the raspi hangs. Had to reboot by switching the power
